clang/lib/CodeGen/CodeGenFunction.h

```diff
@@ -201,14 +211,26 @@ template <> struct DominatingValue<RValue> {
   class saved_type {
     enum Kind { ScalarLiteral, ScalarAddress, AggregateLiteral,
                 AggregateAddress, ComplexAddress };
-
-    llvm::Value *Value;
-    llvm::Type *ElementType;
+    union {
+      struct {
+        DominatingLLVMValue::saved_type first, second;
+      } Vals;
+      DominatingValue<Address>::saved_type AggregateAddr;
+    };
     LLVM_PREFERRED_TYPE(Kind)
     unsigned K : 3;
-    unsigned Align : 29;
-    saved_type(llvm::Value *v, llvm::Type *e, Kind k, unsigned a = 0)
-      : Value(v), ElementType(e), K(k), Align(a) {}
+    unsigned IsVolatile : 1;
```

Comment:
Hi @ahatanak, static verifier is concerned that IsVolatile is not initialized in the constructor. Can we add an initializer here? Also one of the constructors below has a IsVolatile parameter but isn't using it to initialize the field. Do you know what we should be doing here?

Reply:
Prior to this commit, `DominatingValue<RValue>` didn't have any information about whether the saved `RValue` was volatile. I don't think we need that information.

`IsVolatile` should be removed.

Reply:
CC @rjmccall

Reply:
Fixed in https://github.com/llvm/llvm-project/pull/95165